nixos/modules/services/web-apps/healthchecks.nix

```diff
@@ -0,0 +1,216 @@
+{ config, lib, pkgs, buildEnv, ... }:
+
+with lib;
+
+let
+  cfg = config.services.healthchecks;
+  staticDir = cfg.dataDir + "/static";
+  boolToPython = b: if b then "True" else "False";
+  localSettingsFile = pkgs.writeTextFile {
+    name = "local_settings.py";
+    text = ''
+      STATIC_ROOT = '${staticDir}'
+      ALLOWED_HOSTS = ${builtins.toJSON cfg.allowedHosts}
+      # Default database engine is SQLite. So one can just check out code,
+      # install requirements.txt and do manage.py runserver and it works
+      DEFAULT_AUTO_FIELD = "django.db.models.AutoField"
+      DATABASES = {
+          "default": {
+          "ENGINE": "django.db.backends.sqlite3",
+          "NAME": "${cfg.dataDir}/healthchecks.sqlite",
+        }
+      }
```

Comment:
Can I later overwrite this with postgres?

Reply:
Yes, anything in extraConfig will overwrite those default settings